core/src/main/java/com/backblaze/b2/client/B2StorageClient.java

```diff
@@ -211,6 +212,16 @@ default B2Bucket getBucketOrNullByName(String name) throws B2Exception {
      */
     B2FileVersion uploadSmallFile(B2UploadFileRequest request) throws B2Exception;
 
+    /**
+     * Makes a copy of a file in the same bucket.
+     * The new file must be smaller than the maximum file size (5 GB).
+     *
+     * @param request describes what file to copy, the range of bytes to copy, and how to handle file metadata.
+     * @return The B2FileVersion of the new file.
+     * @throws B2Exception if there's any trouble.
+     */
+    B2FileVersion copySmallFile(B2CopyFileRequest request) throws B2Exception;
```

Comment:
Do we use the term "small file" elsewhere?  I'm not sure we have an official name for non-large files.

(The docs seem to use "normal file".)

Reply:
This interface provides an `uploadSmallFile()` method, so I was basing off of that. Searching for "smallFile" in the code, it shows up a bunch, so I'm inclined to keep it as is, for consistency's sake.